src/schema/v2/me/artistTypeUserInterests.ts

```diff
@@ -0,0 +1,53 @@
+import { GraphQLFieldConfig, GraphQLInt } from "graphql"
+import { connectionFromArraySlice } from "graphql-relay"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { pageable } from "relay-cursor-paging"
+import { ResolverContext } from "types/graphql"
+import { createPageCursors } from "../fields/pagination"
+import {
+  userInterestCategoryEnum,
+  UserInterestConnection,
+} from "../userInterests"
+
+export const artistTypeUserInterests: GraphQLFieldConfig<
```

Comment:
This should probably just be an `interestsConnection` which lives under `me`. The type of this field (`UserInterestConnection`) returns a connection with nice edge fields, and where the node field is a union of `Gene` or `Artist`.